dashboard/app/views/levels/instructions.haml

```diff
@@ -0,0 +1,26 @@
+-#
+  This is a view for rendering a summary of all of the levels/instructions in a given
+  script, that should only be visible in levelbuilder mode
+
+- stages = local_assigns[:stages]
+
+- stages.each_with_index do |stage, index|
+  %h1{:style => "margin-top: 30px"}= "Stage #{(index + 1)}: #{stage.name}"
```

Comment:
Prefer moving inline styles to a stylesheet, but if this is truly a one-off page I could maybe be convinced to leave them as inline.